{{cookiecutter.repo_name}}/.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
I can't remember anymore, didn't GitHub switch to a different system recently for all this? I know we wanted to change things in the core repo but I keep forgetting to raise the issue.

Reply:
Yeah I guess we can do yaml now? It looks nicer but doesn't super add much. I'll have a look at it

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

Reply:
Raised https://github.com/MDAnalysis/cookiecutter-mda/issues/14 to deal with this later.